src/main/kotlin/com/faforever/userservice/controller/OAuthController.kt

```diff
@@ -114,4 +116,17 @@ class OAuthController(
         }.flatMap { redirectUrl ->
             redirect(response, redirectUrl)
         }
+
+    @GetMapping("banned")
+    fun showBan(
+        request: ServerHttpRequest,
+        @RequestParam("reason") reason: String,
+        @RequestParam("expiration") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) expiration: LocalDateTime?,
```

Comment:
LocalDateTime is a bad idea because it won't match the users timezone. Rather use OffsetDateTime and then convert it to the users timezone or show UTC.

Reply:
Sounds good. Shall I go ahead and change the type in the Ban object as well?

Reply:
Hmm r2dbc doesn't like that. I wonder how hibernate in the api handles it

Reply:
Done